mesa/examples/basic/boltzmann_wealth_model/app.py

```diff
@@ -58,20 +69,20 @@ def post_process(ax):
 # It will automatically update and display any changes made to this file
 page = SolaraViz(
     model,
-    components=[SpaceGraph, GiniPlot],
+    renderer,
+    components=[GiniPlot],
     model_params=model_params,
     name="Boltzmann Wealth Model",
 )
 page  # noqa
 
 
 # In a notebook environment, we can also display the visualization elements directly
```

Comment:
Do we need this I think this was just carried over form the tutorial but is not relevant to running from the command line.